autotest/test_parameter_read.py

```diff
@@ -1,27 +1,70 @@
-import os
-import sys
-
 import pytest
 
 from pynhm.utils import PrmsParameters
 from utils import assert_or_print
 
 
+@pytest.fixture
+def canopy_parameters():
+    return tuple(("unknown", "srain_intcp", "wrain_intcp", "snow_intcp"))
+
+
 def test_parameter_read(domain):
     parameter_file = domain["param_file"]
     print(f"parsing...'{parameter_file}'")
 
-    p_parameters = PrmsParameters(parameter_file)
-    dimensions = p_parameters.get_dimensions
-    # param_data = p_parameters.get_parameter_data
+    parameters = PrmsParameters.load(parameter_file)
```

Comment:
Do you think it should load on __init__ if the paramter_file arg is supplied?

Reply:
No. The base `__init__` takes a dictionary. This will allow to set parameters directly with a dictionary.